src/cryptoadvance/specter/templates/components/price_bar.jinja

```diff
@@ -203,27 +203,10 @@
             }
             let url = `{{ url_for('price_endpoint.toggle') }}`;
             var formData = new FormData(document.getElementById('price-form'));
-            try {
-                const response = await fetch(
-                    url,
-                    {
-                        method: 'POST',
-                        body: formData
-                    }
-                );
-                if(response.status != 200){
-                    showError(await response.text());
-                    return;
-                }
-                const jsonResponse = await response.json();
-                if (jsonResponse.success) {
-                    location.reload();
-                    return
-                };
-                showError(`{{ _("Failed to toggle price mode...") }}`)
-            } catch(e) {
-                console.log("Caught error: ", e);
-                showError(e);
+            result = await send_request(url, 'POST', "{{ csrf_token() }}", formData);
```

Comment:
```suggestion
            let result = await send_request(url, 'POST', "{{ csrf_token() }}", formData);
```

Reply:
Fixed in https://github.com/cryptoadvance/specter-desktop/pull/2014/commits/f354b0b10e1e4c817dce3e044f9e3c5a815f0dc3